Formula/marcli.rb

```diff
@@ -0,0 +1,28 @@
+class Marcli < Formula
+  desc "Parse MARC (ISO 2709) files"
+  homepage "https://github.com/hectorcorrea/marcli"
+  url "https://github.com/hectorcorrea/marcli/archive/refs/tags/1.0.1.tar.gz"
+  sha256 "abea9ae6a7cd2d6874ac71d0aa429b9eab367daa4f301161a0671f85608216cf"
+  license "MIT"
+  head "https://github.com/hectorcorrea/marcli"
```

Comment:
Minor change for consistency and to ensure `--HEAD` installs use the correct branch.

```suggestion
  head "https://github.com/hectorcorrea/marcli.git", branch: "main"
```